src/styles/blocks/stars.scss

```
@@ -0,0 +1,23 @@
+
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  & div:not(:last-child) {
+    margin-right: 4px;
+  }
```

don't use tag selectors for styling, use class selectors
```suggestion
  &__star:not(:last-child) {
    margin-right: 4px;
  }
```

---

fixed